grug/testing/tests/indexing.rs

```diff
@@ -0,0 +1,149 @@
+use {
+    assertor::*,
+    grug_app::Indexer,
+    grug_testing::TestBuilder,
+    grug_types::{Coins, Denom, Message},
+    indexer_sql::entity,
+    sea_orm::{EntityTrait, QueryOrder},
+    std::str::FromStr,
+};
+
+#[test]
+fn index_block_with_blocking_indexer() {
+    let denom = Denom::from_str("ugrug").unwrap();
+    let mut indexer =
+        indexer_sql::blocking_indexer::BlockingIndexer::new_with_database("sqlite::memory:")
+            .expect("can't create indexer");
+    indexer.start().expect("Can't start indexer");
+    let (mut suite, mut accounts) = TestBuilder::new_with_indexer(indexer.clone())
+        .add_account("owner", Coins::new())
+        .add_account("sender", Coins::one(denom.clone(), 30_000).unwrap())
+        .set_owner("owner")
+        .build();
+
+    let to = accounts["owner"].address;
+
+    let _outcome = suite.send_message_with_gas(
+        &mut accounts["sender"],
+        2000,
+        Message::transfer(to, Coins::one(denom.clone(), 2_000).unwrap()).unwrap(),
+    );
```

Comment:
Use `should_success` from `ResultExt` trait instead to catch in a var
```suggestion
   suite.send_message_with_gas(
        &mut accounts["sender"],
        2000,
        Message::transfer(to, Coins::one(denom.clone(), 2_000).unwrap()).unwrap(),
    ).should_success();
```

Reply:
I copied that code over from existing test (but looking at it it looks like this was fixed in other code too).